src/widgets/OnePressButton.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Button } from 'react-native-ui-components';
+
+export function OnePressButton(props) {
```

Comment:
Cool. Might be a useful thing just to add in react-native-ui-components that we can turn on as a prop. Not now though.

Reply:
Yeah that would be better: https://github.com/sussol/mobile/issues/606

Reply:
Hmmm I can review that and merge it and you can quickly change this PR to use it, sound good?

Reply:
Jokes that's an issue num not a PR num.